fusions.wdl

```diff
@@ -108,7 +109,67 @@ task fusions {
     }
 }
 
-# TODO: Deletion detection task
+# CNV - Indexcov
+task indexcov {
+    input {
+        File full_wgs_idx
+        File ref_fa_fai
+    }
+    Int cores = 1
+    Float idx_size = size([full_wgs_idx], "GB")
+    Int runtime_size = 4 + round(idx_size)
+    runtime {
+        memory: "4GB"
+        cpu: cores
+        preemptible: 1
+        docker: "quay.io/biocontainers/goleft:0.2.4--0"
+        disks: "local-disk ~{runtime_size} SSD"
+        bootDiskSizeGb: runtime_size
+    }
+    command <<<
+        goleft indexcov --extranormalize -d indexcov_out --fai ~{ref_fa_fai} ~{full_wgs_idx}
+        tar -cvf indexcov.tar indexcov_out/
+        gzip indexcov.tar
+    >>>
+    output {
+        File indexcov_out = "indexcov.tar.gz"
+    }
+}
+
+# CNV - mosdepth
+task mosdepth {
+    input {
+        File ref_fa
+        File ref_fa_fai
+        File full_wgs
+        File full_wgs_idx
+        File dnmt3a_mosdepth
+    }
+    Int cores = 1
+    Float bam_size = size([full_wgs], "GB")
+    Int runtime_size = 4 + round(bam_size)
+    runtime {
+        memory: "8GB"
+        cpu: cores
+        preemptible: 1
+        docker: "quay.io/biocontainers/mosdepth:0.2.5--hb763d49_0"
+        disks: "local-disk ~{runtime_size} SSD"
+        bootDiskSizeGb: runtime_size
+    }
+    command <<<
+        ln -s ~{full_wgs} full.cram
+        ln -s ~{full_wgs_idx} full.cram.crai
+        ln -s ~{ref_fa} ref.fa
+        ln -s ~{ref_fa_fai} ref.fa.fai
+        mkdir mosdepth
+        mosdepth -b ~{dnmt3a_mosdepth} -n -f ref.fa mosdepth/cnv full.cram
+        tar -cvf mosdepth.tar mosdepth/
+        gzip mosdepth.tar
+    >>>
+    output {
+        File mosdepth_out = "mosdepth.tar.gz"
+    }
+}
 
 # Workflow to call samtools_fusions and fusions
 workflow runfusions {
```

Comment:
minor quibble, but maybe since it's doing more than fusions now, it should have a new name?

Reply:
I'll call it fusions_mutations for now!